src/libtexture/imagecache.cpp

```diff
@@ -1168,6 +1162,36 @@ ImageCacheFile::mark_broken(string_view error)
 
 
 
+ImageCacheFile*
+ImageCacheImpl::find_file_no_add(ustring filename,
+                                 ImageCachePerThreadInfo* thread_info)
+{
+    // Shortcut - check the per-thread microcache before grabbing a more
+    // expensive lock on the shared file cache.
+    ImageCacheFile* tf = thread_info->find_file(filename);
+
+    // Check the man file cache. For this part, we need to lock the file cache.
```

Comment:
Check the main.
You could check the man, but I'm sure he's fine.

Reply:
fixing, thanks